scipy/stats/_stats_mstats_common.py

```diff
@@ -66,28 +71,32 @@ def linregress(x, y=None):
 
     Perform the linear regression:
 
-    >>> slope, intercept, r_value, p_value, std_err = stats.linregress(x, y)
-    >>> print("slope: %f    intercept: %f" % (slope, intercept))
-    slope: 1.944864    intercept: 0.268578
+    >>> res = stats.linregress(x, y)
 
-    To get coefficient of determination (R-squared):
+    Coefficient of determination (R-squared):
 
-    >>> print("R-squared: %f" % r_value**2)
+    >>> print(f"R-squared: {res.rvalue**2:.6f}")
     R-squared: 0.735498
 
     Plot the data along with the fitted line:
 
     >>> plt.plot(x, y, 'o', label='original data')
-    >>> plt.plot(x, intercept + slope*x, 'r', label='fitted line')
+    >>> plt.plot(x, res.intercept + res.slope*x, 'r', label='fitted line')
     >>> plt.legend()
     >>> plt.show()
 
-    Example for the case where only x is provided as a 2x2 array:
+    Calculate 95% confidence interval on slope and intercept:
 
-    >>> x = np.array([[0, 1], [0, 2]])
-    >>> r = stats.linregress(x)
-    >>> r.slope, r.intercept
-    (2.0, 0.0)
+    >>> # Two-sided inverse Students t-distribution
+    >>> # p - probability, df - degrees of freedom
+    >>> from scipy.stats import t
+    >>> tinv = lambda p,df: abs(t.ppf(p/2,df))
+
+    >>> ts = tinv(0.05,len(x)-2)
+    >>> print(f"slope (95%%): {res.slope:.6f} +/- {ts*res.stderr:.6f}")
+    slope (95%): 1.944864 +/- 0.950885
+    >>> print(f"intercept (95%%): {res.intercept:.6f} +/- {ts*res.intercept_stderr:.6f}")
```

Comment:
I'd prefer something like:
```
>>> print(f"intercept (95%): {res.intercept:.6f}"
    ...   f" +/- {ts*res.intercept_stderr:.6f}")
```
Since it's clearer, consistent with the previous print and does not introduce an unnecessary variable.